packages/pwa-kit-dev/src/configs/webpack/overrides-plugin.js

```diff
@@ -59,9 +59,14 @@ class OverridesResolverPlugin {
         })
     }
     /**
+     * This takes an import with the overrides dir / extends package removed and finds
+     * the corresponding path from overridesFsRead
      *
-     * @param requestPath
-     * @param dirs
+     * Ie. Given an import @salesforce/retail-react-app/app/components/header,
+     * requestPath is app/components/header
+     *
+     * @param requestPath - relative import path
+     * @param dirs - this._allSearchDirs
      */
     findFileFromMap(requestPath, dirs) {
```

Comment:
dirs is always `this._allSearchDirs`. We can just read the value of `this._allSearchDirs` in here and remove `dirs` as an input